docs/getting-started-on-bdp/private-managed-cloud/setup-guide-azure/index.md

```diff
@@ -78,6 +78,19 @@ Create a custom role and assign it the “Snowplow BDP Enterprise Deployment”
 ### Determine if Snowplow requires a specific VPC (optional)
 If you require Snowplow to be deployed into a specific VPC CIDR range, this should be provided at the same time as credentials. We need a /16 provided for the VPC so that we can create /20 subnets (note: [VPC peering and using a custom VPC is an additional bolt-on](https://snowplow.io/snowplow-behavioral-data-platform-product-description/#h-vpc-peering-aws-gcp))
 
+### Check subscription resource availablity
+You must check that the following infrastructure is available in the subscription before deplyoment starts. If the required instances are unavailable, it may result in delays to getting started. 
```

Comment:
We can and have used other instance types.  So the list here isn't an absolute list that we can only deploy into. We want to be careful that we don't appear to limit ourselves.  If we get a much higher volume customer, then its much more cost effective to have instance types which are double or triple this size.  We do same in AWS and GCP.  We aren't accounting for this here.